agent/app/agent.go

```diff
@@ -751,6 +747,37 @@ func (agent *ecsAgent) constructVPCSubnetAttributes() []*ecs.Attribute {
 	}
 }
 
+// Loads Managed Daemon images for all Managed Daemons registered on the Agent.
+// The images are loaded in the background. Successfully loaded images are added to
+// imageManager's cleanup exclusion list.
+func (agent *ecsAgent) loadManagedDaemonImagesAsync(imageManager engine.ImageManager) {
+	daemonManagers := agent.getDaemonManagers()
```

Comment:
Just curious, have we tried using the the existing `GetDaemonManagers` in our task engine? Curious if we could just use that instead. https://github.com/aws/amazon-ecs-agent/blob/dev/agent/engine/docker_task_engine.go#L1230

Reply:
That's a good find! I wasn't aware of that. 

It seems that the underlying map pointed to by `ecsAgent.daemonManagers` and `engine.daemonManagers` is the same, so it doesn't matter which one we use here. 
https://github.com/aws/amazon-ecs-agent/blob/dev/agent/app/agent.go#L346-L348

I prefer using the one from `ecsAgent` type as the image load method is added to `ecsAgent` type. Does that sound okay?

Reply:
yep yep sounds good to me!

Reply:
Thanks!